examples/CMakeLists.txt

```diff
@@ -64,6 +63,14 @@ else()
     message(STATUS "No Kokkos found, disabling examples with Kokkos assembly.")
 endif()
 
+set(GKO_FILE_CONFIG_ENABLED 0)
```

Comment:
I think if you also add GINKGO_BUILD_EXAMPLES in the third_party CMakeLists, you do not need the change here

Reply:
Then we always pull in nlohmann_json when examples is built. But I guess that is okay.